.gitignore

```diff
@@ -25,4 +25,7 @@ yarn-error.log*
 # django stuff
 *.env
 *.pyc
-*.sqlite3
\ No newline at end of file
+*.sqlite3
+
+#mysql stuff
```

Comment:
How is vscode "#mysql stuff"? Could you provide either an appropriate comment or no comment at all.